src/newViews/Tagging/Sources/BulkUpload/Upload/FilesUpload/styles.scss

```diff
@@ -6,6 +6,8 @@
         display: flex;
         flex-direction: column;
         flex-grow: 1;
+        border: none;
+        --height-border: 0;
```

Comment:
Is this used?

Reply:
Yes, this is for overwriting the border of input under this container.
@frozenhelium not sure if I should do this.